environment/create_google_config.py

```diff
@@ -0,0 +1,378 @@
+    while request is not None:
+
+        # execute the request
+        response = request.execute()
+
+        # add projects to the projects list
+        if 'billingAccounts' in response:
+            billing_accounts_list.extend(response['billingAccounts'])
+
+        request = billing_accounts.list_next(request, response)
+
+    if len(billing_accounts_list) == 0:
+    	# User does not have access to any billing accounts
+    	print "You do not have a Google billing account set up. In order to run\nWDLs in the Google cloud you need an account to bill to. See the README\nfor more details.\nTo learn about creating a billing account, see here: \nhttps://cloud.google.com/billing/docs/how-to/manage-billing-account#create_a_new_billing_account"
+        sys.exit("Exiting.")
+
+    else:
+    	print "\nYou have access to the following Google billing accounts: "
+    	
+    	# Setup table
+    	headers = "Billing Account ID\tBilling Account Name"
+    	print headers
+    	print '-' * len(headers.expandtabs()) 
+
+    	# Iterate and print every billing account 
+    	for billing_acct in billing_accounts_list:
+    		print "%s\t%s" % (billing_acct["name"].replace("billingAccounts/",""), billing_acct["displayName"])
+
+    	print "\nEnter the \"Billing Account ID\" of the billing account you want to use\nto create a new Google project. This will be the billing account that is charged\nfor storage and compute costs."
+    	ex_billing_acct = "002481-B7351F-CD111E"
+    	billing_account_id = raw_input("(IDs are case-sensitive and will look similar to this: %s): " % ex_billing_acct)
+    	while len(billing_account_id) != len(ex_billing_acct):
+			billing_account_id = raw_input("Please enter a valid billing account: ")
+    	print "\nYou have selected this Billing Account: %s" % billing_account_id
+
+    	# Project name with datetime stamp (minute and second) and user's email address
+    	# DO NOT PUT AN UNDERSCORE '_' IN THE NAME, it cannot be longer than 30 characters, nor can it have "google"
+    	user_name = check_output(['gcloud', 'config', 'get-value', 'core/account']).rstrip().partition("@")[0]
+    	project_name = "cromwell-%s" % user_name + datetime.datetime.now().strftime("-%M-%S")
+    	create_google_project(project_name, billing_account_id)
+    	return project_name
+
+# Create a google project for the user
+def create_google_project(project_name, billing_account_id):
+	# Create google project
+	body = {'project_id': '%s' % project_name, 'name': '%s' % project_name, 'labels': None}
+	crm.projects().create(body=body).execute()
+	
+	# Check the project is ready
+	check_project_created(project_name)
+	
+	# Link new project to billing account
+	enable_billing_account(billing_account_id, project_name)
+	return project_name
+
+def check_project_created(project_name):
+	# List projects currently created
+	result = crm.projects().list().execute()
+
+	# Search through list of services to see if the API has been enabled  
+	while True: 
```

Comment:
later on we will need error handling to break out of these loops if the project fails to be created (same for enabling APIs, billing, etc)